src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -2205,6 +2206,94 @@ public static bool FoundInUnboundLambda(BoundNode methodBody, IdentifierNameSynt
                 return base.Visit(node);
             }
         }
+
+        private sealed class WasPropertyBackingFieldAccessChecked : BoundTreeWalkerWithStackGuardWithoutRecursionOnTheLeftOfBinaryOperator
+        {
+            public static BoundPropertyAccess? FindUncheckedAccess(BoundNode node)
+            {
+                var walker = new WasPropertyBackingFieldAccessChecked();
+                walker.Visit(node);
+                return walker._found;
+            }
+
+            private BoundPropertyAccess? _found;
+            private bool _suppressChecking;
+
+            private WasPropertyBackingFieldAccessChecked()
+            {
+            }
+
+            public override BoundNode? Visit(BoundNode? node)
+            {
+                if (_found is { })
+                {
+                    return node;
```

Comment:
Is this return value used?

Reply:
No, the return value is not used, and in fact most of the visit methods return `null`. I'll change this to `return null;`.